tests/simple/testdata/broken.textproto

```diff
@@ -61,3 +61,31 @@ section {
     }
   }
 }
+section {
+  name: "integer _math.textproto/Literal Integer Math"
+  description: "tests for integer math."
+  test {
+    name: "int64_overflow"
+    expr: "9223372036854775807 + 1"
```

Comment:
Add a "description" field that explains this magic number, such as "INT64_MAX plus one."  Do this for all the tests.